### ISSUE TITLE
Will libcudacxx conflict against GNU C++'s header files?

### ISSUE THREAD

**Issue by jiapei100** [NVIDIA/thrust#1695]
Will libcudacxx conflict against GNU C++'s header files?

**Comment by jrhemstad** [comment]
No. libcu++ requires explicit opt-in and lives alongside the host standard library. The headers exist in a distinct directory `<cuda/std/atomic>` or `<cuda/atomic>` and namespaces are similarly unambiguous for `cuda::std::atomic` or `cuda::atomic`. 

See https://youtu.be/-ENnYEWezKo?t=319
